### PR TITLE
Initial rock paper scissor upload

### PR DESCRIPTION
**Pull request by Reklund3**

### CODE REVIEW

File: RockPaperScissors/RockPaperScissors.cs [hunk 1]
```diff
@@ -6,20 +6,100 @@ class Program
...
+            string playerHand = System.String.Empty;
+            string compHand = System.String.Empty;
+            string[] compOptions = { "rock", "paper", "scissors" };
+            // Main game loop, continues until the getContinue function returns a false
+            do
+            {
+                playerHand = getHand(compOptions);
+                compHand = compOptions[rand.Next(0,3)];
+                Console.WriteLine(compHand);
+                Console.WriteLine(CompareHands(playerHand, compHand, ref playerScore, ref computerScore));
```
Comment (yousifS) [inline RockPaperScissors/RockPaperScissors.cs]: A different approach wold have been for the method to return 1 if the first hand won, 2 if the second hand won, or 0 if neither won.

File: RockPaperScissors/RockPaperScissors.cs [hunk 2]
```diff
@@ -6,20 +6,100 @@ class Program
...
-            return hand1 + ' ' + hand2;
+            Console.WriteLine("Please enter Rock, Paper, or Scissors:");
+            string playerHand = Console.ReadLine().ToLower();
+            for( int i = 0; i < compOptions.Length; i++)
+            {
+                if (playerHand == compOptions[i])
+                {
+                    return playerHand;
+                }   
+            }
```
Comment (yousifS) [inline RockPaperScissors/RockPaperScissors.cs]: nice use of recursion.

File: RockPaperScissors/RockPaperScissors.cs (hunk 1 above)
Comment (Reklund3) [reply RockPaperScissors/RockPaperScissors.cs]: Yeah, I need to do something to clean that function call up. Additionally, i will have to change the function to an int to return and int value and change all the comments to actual console.writelines. maybe i should do that on the current RPS game with the try catch.

File: RockPaperScissors/RockPaperScissors.cs (hunk 2 above)
Comment (Reklund3) [reply RockPaperScissors/RockPaperScissors.cs]: awesome, thank you. will have to review the implementation of this over the other recursive calls and see whats better about this implementation.

**Pull request merged**